manifest.yml

```diff
@@ -0,0 +1,3 @@
+applications:
```

Comment:
I think we may need a few environment variables defined ( as in config.ini ) ? i.e an env: section in the manifest.yml